projects/packages/blaze/src/js/editor.js

```diff
@@ -50,6 +50,11 @@ const BlazePostPublishPanel = () => {
 	} );
 
 	const { tracks } = useAnalytics();
+	useEffect( () => {
+		if ( isPostPublished ) {
```

Comment:
This isn't enough, it gets triggered when you open the editor for a post that's already published. I need to fix that.

Reply:
e4a0745801c461f5f116d85adf0aa1acb6b71925 should help with that